### PR TITLE
toString Method Pull Request (Issue #4)

### PR DESCRIPTION
**Pull request by danindugit**
I have made the toString methods for the Chunk class and all of its sub-classes. I have also indented the files that I made changes in using the standard eclipse indenting. This is regarding Issue #4 (toString methods on chunks).

### CODE REVIEW

File: src/main/java/png/chunk/Chunk.java [hunk 1]
```diff
@@ -12,56 +12,63 @@
...
-        return DATA_LENGTH_SIZE + TYPE_SIZE + dataLength + CRC_SIZE;
-    }
+	public AllowedChunkTypes getType() {
+		return type;
+	}	
 
-    public AllowedChunkTypes getType() {
-        return type;
-    }
+	//toString method for Chunk class
```
Comment (IgnacioRondini) [inline src/main/java/png/chunk/Chunk.java:68]: There is no need to comment here, the toString() method is self explanatory :)

File: src/main/java/png/chunk/IHDRChunk.java [hunk 2]
```diff
@@ -1,66 +1,78 @@
...
+	public byte getFilterMethod() {
+		return filterMethod;
+	}
+
+	public byte getInterlaceMethod() {
+		return interlaceMethod;
+	}
+
+	//toString method for IHDRChunk Class
+	public String toString() {
```
Comment (IgnacioRondini) [inline src/main/java/png/chunk/IHDRChunk.java:69]: Note that here we lost the information about the toString method in the chunk class. We could for instance call super.ToString and concatenate it with what you add.

File: src/main/java/png/chunk/IENDChunk.java [hunk 3]
```diff
@@ -1,13 +1,22 @@
...
 
 public class IENDChunk extends Chunk {
-    IENDChunk(ByteBuffer buffer) {
-        super(buffer);
-    }
+	IENDChunk(ByteBuffer buffer) {
+		super(buffer);
+	}
+
+	//toString method for IENDChunk Class
```
Comment (IgnacioRondini) [inline src/main/java/png/chunk/IENDChunk.java:15]: No need to  duplicate this, as super.string() does the same :)

File: src/main/java/png/chunk/Chunk.java (hunk 1 above)
Comment (danindugit) [reply src/main/java/png/chunk/Chunk.java:68]: You're right. I will remove the comments.

File: src/main/java/png/chunk/IENDChunk.java (hunk 3 above)
Comment (danindugit) [reply src/main/java/png/chunk/IENDChunk.java:15]: I removed the repeated code :)

File: src/main/java/png/chunk/IHDRChunk.java (hunk 2 above)
Comment (danindugit) [reply src/main/java/png/chunk/IHDRChunk.java:69]: I called super.toString() and added it to the beginning of this toString().

File: src/main/java/png/chunk/IDATChunk.java [hunk 4]
```diff
@@ -13,7 +13,12 @@
...
 public class IDATChunk extends Chunk {
-    IDATChunk(ByteBuffer buffer) {
-        super(buffer);
-    }
+	IDATChunk(ByteBuffer buffer) {
+		super(buffer);
+	}
+
+	public String toString() {
+		return "IDATChunk [ toString()=" + super.toString() + ", getClass()="
```
Comment (IgnacioRondini) [inline src/main/java/png/chunk/IDATChunk.java:21]: Perfect, I would then write `return "IDATChunk [ super.toString()=" + super.toString() +" ]"` to be coherent with the other your other method. I think that there is no need to add the getClass and hashcode.

Comment (danindugit) [reply src/main/java/png/chunk/IDATChunk.java]: I have made that change accordingly :)

**Pull request merged**